### PR TITLE
feature

### PR DESCRIPTION
**Pull request by BubbleSci**
##### Checklist

- [ ] `npm run test` passes
- [ ] tests and/or benchmarks are included
- [ ] cases or donate is changed or added
- [ ] documentation is changed or added

### CODE REVIEW

File: packages/babel-plugin-import-regenerator/README_CN.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Install regenerator-runtime dependence
+$ npm install regenerator-runtime --save
+
+# Install babel plugin 
+$ npm install @wepy/babel-plugin-import-regenerator --save-dev
+```
+
+## 用法
+
+在您的wepy.config.js中放入一下内容:
```
Comment (Gcaufy) [inline packages/babel-plugin-import-regenerator/README_CN.md:17]: 不需要使用主谓宾, 英文与中文之间加入空格

File: packages/babel-plugin-import-regenerator/README_CN.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        ],
+        plugins: [
+          '@wepy/babel-plugin-import-regenerator'
+        ]
+      }
+    }
+}
+```
+
+## 执照
```
Comment (Gcaufy) [inline packages/babel-plugin-import-regenerator/README_CN.md:37]: License 不用翻译

File: packages/redux/README_CH.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+```
+// app.wpy
+import wepy from '@wepy/core';
+import wepyRedux from '@wepy/redux';
+
+wepy.use(wepyRedux);
+```
+
+2. 初始化商店
```
Comment (Gcaufy) [inline packages/redux/README_CH.md:21]: 翻译有误

**Comment by BubbleSci** [comment]
好的收到 我更改一下




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Gcaufy"<notifications@github.com&gt;;
发送时间:&nbsp;2020年7月4日(星期六) 中午11:39
收件人:&nbsp;"Tencent/wepy"<wepy@noreply.github.com&gt;;
抄送:&nbsp;"带鱼希儿呀"<1228160604@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [Tencent/wepy] feature (#2632)





 
@Gcaufy requested changes on this pull request.
 
 
In packages/babel-plugin-import-regenerator/README_CN.md:
 &gt; + +允许wepy使用 `Async Functions`. + +## 安装 + +``` +# Install regenerator-runtime dependence +$ npm install regenerator-runtime --save + +# Install babel plugin  +$ npm install @wepy/babel-plugin-import-regenerator --save-dev +``` + +## 用法 + +在您的wepy.config.js中放入一下内容:  
不需要使用主谓宾, 英文与中文之间加入空格
 
 
In packages/babel-plugin-import-regenerator/README_CN.md:
 &gt; +{ +  .... +    compilers: { +      babel: { +        presets: [ +          '@babel/preset-env' +        ], +        plugins: [ +          '@wepy/babel-plugin-import-regenerator' +        ] +      } +    } +} +``` + +## 执照  
License 不用翻译
 
 
In packages/redux/README_CH.md:
 &gt; +npm install @wepy/redux redux --save +``` + +## 用法 + +1. 安装Redux + +``` +// app.wpy +import wepy from '@wepy/core'; +import wepyRedux from '@wepy/redux'; + +wepy.use(wepyRedux); +``` + +2. 初始化商店  
翻译有误

**Comment by Gcaufy** [comment]
其它问题：
1. 文件名不统一， 有些 _CH.md， 有些 _CN.md
2. 需要添加语言文档跳转的 link
3. commit msg 不符合规范

File: packages/babel-plugin-import-regenerator/README_CN.md (hunk 1 above)
Comment (weimingxinas) [inline packages/babel-plugin-import-regenerator/README_CN.md:17]: 一下  以下

File: packages/plugin-define/README_CH.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# wepy定义插件
```
Comment (weimingxinas) [inline packages/plugin-define/README_CH.md:1]: 这个也不需要翻译

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2632) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2632) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/wepy?pullRequest=2632) it.</sub>

**Pull request closed without merging**